src/functions/images.js

```diff
@@ -0,0 +1,9 @@
+const { Function, config, util: { fetch } } = require("../index");
+
+module.exports = class extends Function {
+
+    run(endpoint, load = {}) {
+        return fetch(`https://server.pengubot.com/${endpoint}`, { method: "POST", headers: { authorization: config.apis.pengu }, body: JSON.stringify(load) }, "buffer");
```

Comment:
use this.fetchURL.
body doesnt need to be stringified. set Content-Type if sending json